docs/INSTALL.md

```diff
@@ -119,20 +118,40 @@ To configure/install the UPF in DPDK mode, the following changes are required:
      # In the order of (n-s1u n-sgi)
     -nhmacaddrs=(22:53:7a:15:58:50 22:53:7a:15:58:50)
     +nhmacaddrs=(b4:96:91:b4:47:b8 b4:96:91:b4:47:b9)
-
      # IPv4 route table entries in cidr format per port
      #
     @@ -111,7 +111,7 @@ sudo rm -rf /var/run/netns/pause
      make docker-build
-
      if [ "$mode" == 'dpdk' ]; then
     -       DEVICES=${DEVICES:-'--device=/dev/vfio/48 --device=/dev/vfio/49 --device=/dev/vfio/vfio'}
     +       DEVICES=${DEVICES:-'--device=/dev/vfio/184 --device=/dev/vfio/185 --device=/dev/vfio/vfio'}
         PRIVS='--cap-add IPC_LOCK'
-
```

Comment:
Why does this empty line need to be removed? It is part of the output of git diff

Reply:
ok